cmd/wasmd/commands.go

```diff
@@ -260,7 +260,6 @@ var tempDir = func() string {
 	if err != nil {
 		panic("failed to create temp dir: " + err.Error())
 	}
-	defer os.RemoveAll(dir)
```

Comment:
This defer call just immediately removed the folder after being created. This causes wasmvm to create the folder again and then it is never deleted. Over time that leads to a lot of temp folders accumulating when people run the cli.
What we actually want to do is to remove it after we are done with it.